api/include/opentelemetry/trace/propagation/HttpTraceContext.cc

```diff
@@ -0,0 +1,129 @@
+#include <vector>
+
+namespace trace
+{
+namespace propagation
+{
+namespace
+{
+class HttpTraceContext : public HTTPTextFormat
+{
+
+    private:
+        static const nostd::string_view TRACE_PARENT = "traceparent";
+        static const nostd::string_view TRACE_STATE = "tracestate";
+        static const std::vector<nostd::string_view> FIELDS = {TRACE_PARENT, TRACE_STATE)};
```

Comment:
This static vector isn't safe, see https://google.github.io/styleguide/cppguide.html#Static_and_Global_Variables

There are a couple options. One is to keep a pointer to the vector and statically initialize it from a function. Since it's only used in one place, I would move the declaration there and do something like

        List<String> fields() {
            static const auto* FIELDS = new std::vector<nostd::string_view>({TRACE_PARENT, TRACE_STATE});
            return FIELDS;
        }